exporter/googlemanagedprometheusexporter/README.md

```diff
@@ -120,8 +127,59 @@ In the configuration above, `cloud.availability_zone`, `cloud.region`, and
 the `gcp` detector. The prometheus receiver sets `service.name` to the
 configured `job_name`, and `service.instance.id` is set to the scrape target's
 `instance`. The prometheus receiver sets `k8s.namespace.name` when using
-`role: pod`.  The `groupbyattrs` processor promotes `location`, `cluster`, and
-`namespace` labels on metrics to resource labels, which allows overriding (e.g.
-using prometheus metric_relabel_configs) the attributes discovered by other
-receivers or processors. This is useful when metric exporters already have
-`location`, `cluster`, or `namespace` labels, such as Kube-state-metrics.
+`role: pod`.
+
+In GMP, the above attributes are used to identify the `prometheus_target`
+monitored resource. As such, it is recommended to avoid writing metric or resource labels
+that match these keys. Doing so can cause errors when exporting metrics to
+GMP or when trying to query from GMP. So, the recommended way to set them
+is with the [resourcedetection processor](../../processor/resourcedetectionprocessor).
+
+If you still need to set `location`, `cluster`, or `namespace` labels
+(such as when running in non-GCP environments), you can do so with the
+[resource processor](../../processor/resourceprocessor) like so:
+
+```yaml
+processors:
+  resource:
+    attributes:
+    - key: "location"
+      value: "us-east-1"
+      action: upsert
+```
+
+In cases where your metrics already contain labels that conflict with
+the above names, you should rename those labels to avoid conflicting
+with the monitored resource labels. You can do this with the
+[transform processor](../../processor/transformprocessor) like so:
+
+```yaml
+processors:
+  transform:
+    metric_statements:
```

Comment:
Is this still needed?

Reply:
I guess not, but it does explain what's going on with all the transform statements above which I think could be useful if it's not info overload

Reply:
I'd prefer removing it.  Feel free to incorporate some of the language into the comment above, if you think its helpful.

Reply:
Meh I'm not too attached to it, removed